specification/managementgroups/resource-manager/Microsoft.Management/stable/2020-02-01/management.json

```diff
@@ -1787,14 +1787,14 @@
         "type": {
           "type": "string",
           "enum": [
-            "Microsoft.Management/managementGroup"
+            "Microsoft.Management/managementGroups"
```

Comment:
I have to change this, because by design, the type of a resource should be in plural form. This must be a mistake.

Reply:
Can we ask service team to double confirm it is really typo?